augur/routes/contributor_reports.py

```diff
@@ -1058,10 +1079,16 @@ def returning_contributor_pie_chart():
 
     @server.app.route('/{}/contributor_reports/returning_contributors_stacked_bar/'.format(server.api_version),
                       methods=["GET"])
-    def returning_contributor_stacked_bar():
+    def returning_contributors_stacked_bar():
```

Comment:
Same note as above.

Reply:
Yep, it is just a change so the method name matches the current API endpoint path